### DISCUSSION TITLE
What is USE_COMPILED?

### DISCUSSION THREAD

**Discussion by ebrahimebrahim** [Project-MONAI/MONAI#4505] [Q&A] [answered]
Can someone explain what is the meaning of the flag `monai.config.deviceconfig.USE_COMPILED`?

**Comment by Nic-Ma** [comment]
Hi @wyli ,

Could you please help share some info about this question? I don't remember clearly..

Thanks in advance.

**Comment by wyli** [comment, marked as answer]
it's a flag to control whether to use monai's extension [csrc/resample](https://github.com/Project-MONAI/MONAI/tree/dev/monai/csrc/resample) to do the resampling, currently an experimental feature because https://github.com/Project-MONAI/MONAI/issues/789 is not addressed.
